api-audit/src/main/java/com/capitalone/dashboard/ApiSettings.java

```diff
@@ -18,6 +18,7 @@
     private String peerReviewContexts;
     private String peerReviewApprovalText;
     private String serviceAccountOU;
+    private String incrementVersionTagRegEx;
```

Comment:
may be keep it generic? "commitLogIgnoreAuditRegEx"?

Reply:
@tabladrum  done